tests/Nox.Tests/DatabaseIntegrationTests/Models/TestWebAppDbContext.cs

```diff
@@ -101,6 +101,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             var codeGeneratorState = new NoxSolutionCodeGeneratorState(_noxSolution, _clientAssemblyProvider.ClientAssembly);
             foreach (var entity in _noxSolution.Domain.Entities)
             {
+                Console.WriteLine($"TestWebAppDbContext Configure database for Entity {entity.Name}");
```

Comment:
do we want this?

Reply:
just for now... to see how workplace behaves

Reply:
👍